src/mscorlib/src/System/Guid.cs

```diff
@@ -1285,35 +1285,40 @@ public String ToString(String format)
             return ToString(format, null);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static char HexToChar(int a)
         {
             a = a & 0xf;
             return (char)((a > 9) ? a - 10 + 0x61 : a + 0x30);
         }
 
-        unsafe private static int HexsToChars(char* guidChars, int offset, int a, int b)
+        unsafe private static int HexsToChars(char* guidChars, int a, int b)
         {
-            return HexsToChars(guidChars, offset, a, b, false);
```

Comment:
@AndyAyersMS perhaps we want to capture this sequence as a potential test case for future tuning of the inlining hueristics.